src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -88,6 +88,14 @@ export const VMProvider: FunctionComponent = ({ children }) => {
             new BlocklyInterpreter(code, {
               onHighlight: (id) =>
                 dispatch(blocklySlice.actions.highlightBlock({ blockId: id })),
+
+              // dummy implementations
+              onSetDcMotorPower: (port: number, power: number) =>
+                console.log(
```

Comment:
should this be debug level

Reply:
this is a placeholder till we have some way of getting these values to the real simulator

Reply:
i removed it